config/locales/user_mailer/en.yml

```diff
@@ -11,34 +13,26 @@ en:
       'yes': 'yes'
     email_changed:
       help: >
-        If you did not change your email address, please immediately call the
-        %{app} customer contact center at %{contact_phone_number} between 8:00
-        AM and 8:00 PM Eastern Time, Monday through Friday, except federal
-        holidays.
+        If you did not change your email address, please visit the %{app}
+        %{help_link} or %{contact_link}.
       intro: >
         You have asked %{app} to change the email address currently associated
         with your account.
     password_changed:
       help: >
-        If you did not mean to change your password, please immediately call the
-        %{app} customer contact center at %{contact_phone_number} between 8:00
-        AM and 8:00 PM Eastern Time, Monday through Friday, except federal
-        holidays.
+        If you did not change your password, please visit the %{app}
+        %{help_link} or %{contact_link}.
       intro: You have changed your password for your %{app} account.
     phone_changed:
       help: >
-        If you did not mean to change your phone, please immediately call the
-        %{app} customer contact center at %{contact_phone_number} between 8:00
-        AM and 8:00 PM Eastern Time, Monday through Friday, except federal
-        holidays.
+        If you did not change your phone, please visit the %{app} %{help_link}
+        or %{contact_link}.
       intro: You have changed your phone number for your %{app} account.
       subject: Your phone number has been changed
     signup_with_your_email:
       help: >
-        If you did not request a new account or suspect an error, please
-        immediately call the %{app} customer contact center at
-        %{contact_phone_number} between 8:00 AM and 8:00 PM Eastern Time, Monday
-        through Friday, except federal holidays.
+        If you did not request a new account or suspect an error, please visit
+        the %{app} %{help_link} or %{contact_link}.
```

Comment:
I think we talked about this in a past PR and the conclusion was that it is more confusing to extract the text than to just have the duplication.